twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Footer.js

```diff
@@ -39,6 +41,22 @@ export default class Footer extends Component {
             window.open(InitService.server_infos.help_link);
           })
         }
+        onClickFeedback={
+          () => {
+            const currentUser = UserService.getCurrentUser();
+            Globals.window.freddyWidget.show({
```

Comment:
We don't use Globals anymore (it was for ReactNative) you can directly do:
`window.freddyWidget` in js or `(window as any).freddyWidget` in ts